test/fixtures/@typescript-eslint/prefer-nullish-coalescing.fail.ts

```diff
@@ -4,3 +4,6 @@
 /* eslint-disable @typescript-eslint/strict-boolean-expressions */
 let value: string | undefined;
 value || 'fallback';
+
+let foo: string | undefined;
```

Comment:
Added an example of invalid code to demonstrate the new configuration for `ignoreTernaryTests` option